package.json

```diff
@@ -25,9 +25,12 @@
     "node": ">=14.0.0"
   },
   "scripts": {
-    "prebuild": "rimraf dist",
-    "build": "tsc --module commonjs --project tsconfig.build.json && rollup -c rollup.config.ts",
-    "build:local": "rimraf fsxa-api-*.tgz && npm version prerelease --no-git-tag-version --preid alpha && npm run build && npm pack",
+    "prebuild": "rimraf dist && rimraf proxy/dist",
+    "build:remote": "tsc --module commonjs --project tsconfig.build.json",
+    "build:proxy": "tsc --module commonjs --project tsconfig.proxy.json",
+    "build": "tsc --module commonjs --project tsconfig.build.json && tsc --module commonjs --project tsconfig.proxy.json && npm run build:remote && npm run build:proxy && rollup -c rollup.config.ts",
+    "version:dev": "export NEW_VER=$(npm version prerelease --no-git-tag-version --preid alpha); npm version --workspace proxy ${NEW_VER}",
```

Comment:
oh my :-D how to test it?

Reply:
Run `npm run version:dev` and look into the package.json of the Root project  and the proxy workspace. They should have the same, changed version.